erc20/src/lib.rs

```diff
@@ -36,6 +36,9 @@ impl ERC20 {
         if owner != address!("0000000000000000000000000000000000000007") {
             revert();
         }
+        if tx_gas_price() != U256::from(42) {
+            revert();
+        }
```

Comment:
Why this? Just to use it as an example?

Reply:
Yeah, do we want to keep it for unit tests instead?

Reply:
yea I think it'd be better to keep this example as clean as possible

Reply:
Makes sense!